docs/get-started/authentication.md

```diff
@@ -1,206 +1,199 @@
+Most individual Google accounts (free and paid) don't require a
+Google Cloud project for authentication. However, you'll need to set a Google
+Cloud project when you meet at least one of the following conditions:
 
-# Or, using the fallback variable:
-export GOOGLE_CLOUD_PROJECT_ID="YOUR_PROJECT_ID"
-```
+- You are using a Company, School, or Google Workspace account.
+- You are using a Gemini Code Assist license from the Google Developer Program.
+- You are using a license from a Gemini Code Assist subscription.
 
-To make this setting persistent, see
-[Persisting Environment Variables](#persisting-environment-variables).
+For instructions, see [Set your Google Cloud Project](#set-gcp).
 
-### Use Gemini API Key
+## Use Gemini API Key <a id="gemini-api"></a>
 
 If you don't want to authenticate using your Google account, you can use an API
 key from Google AI Studio.
 
-1.  Obtain your API key from
-    [Google AI Studio](https://aistudio.google.com/app/apikey).
-2.  Set the `GEMINI_API_KEY` environment variable:
+To authenticate and use Gemini CLI with a Gemini API key:
+
+1. Obtain your API key from
+  [Google AI Studio](https://aistudio.google.com/app/apikey)
+  
+2. Set the `GEMINI_API_KEY` environment variable to your key. For example:
 
     ```bash
     # Replace YOUR_GEMINI_API_KEY with the key from AI Studio
     export GEMINI_API_KEY="YOUR_GEMINI_API_KEY"
     ```
+    
+    To make this setting persistent, see
+    [Persisting Environment Variables](#persisting-vars).
+
+3. Start the CLI:
+
+    ```bash
+    gemini 
+    ```
 
-To make this setting persistent, see
-[Persisting Environment Variables](#persisting-environment-variables).
+4. Select **Use Gemini API key**.
 
 > **Warning:** Treat API keys, especially for services like Gemini, as sensitive
 > credentials. Protect them to prevent unauthorized access and potential misuse
 > of the service under your account.
 
-### Use Vertex AI
-
-If you intend to use Google Cloud's Vertex AI platform, you have several
-authentication options:
+## Use Vertex AI <a id="vertex-ai"></a>
 
-- Application Default Credentials (ADC) and `gcloud`.
-- A Service Account JSON key.
-- A Google Cloud API key.
+To use Gemini CLI with Google Cloud's Vertex AI platform, choose from the
+following authentication options:
 
-#### First: Set required environment variables
+- A. Application Default Credentials (ADC) using `gcloud`.
+- B. Service account JSON key.
+- C. Google Cloud API key.
 
-Regardless of your method of authentication, you'll typically need to set the
-following variables: `GOOGLE_CLOUD_PROJECT` (or `GOOGLE_CLOUD_PROJECT_ID`) and
-`GOOGLE_CLOUD_LOCATION`.
+Regardless of your authentication method for Vertex AI, you'll need to set
+`GOOGLE_CLOUD_PROJECT` to your Google Cloud project ID with the Vertex AI API
+enabled, and `GOOGLE_CLOUD_LOCATION` to the location of your Vertex AI resources
+or the location where you want to run your jobs.
 
-To set these variables:
+For example:
```

Comment:
I actually prefer to say what a code sample does, usually because people are scanning for them - but I'm willing to be wrong on this.